src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -1056,6 +1053,7 @@ describe("hubConnection", () => {
     function getJwtToken(url: string): Promise<string> {
         return new Promise((resolve, reject) => {
             const httpClient = new DefaultHttpClient(NullLogger.instance);
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
You mean convert the `getJwtToken` func to async and get rid of the inner Promise or make the inner executor async? If the latter, it will run into this: https://eslint.org/docs/rules/no-async-promise-executor. What it's probably better is this:

```
async function getJwtToken(url: string): Promise<string> {
        const httpClient = new DefaultHttpClient(NullLogger.instance);
        const response = await httpClient.get(url);

        if (response.statusCode >= 200 && response.statusCode < 300) {
            return response.content as string;
        } else {
            throw new Error(response.statusText);
        }
    }
```

Just not sure about the throw. What you think?
The functional tests are not running on my box bcs I don't have the preview sdk installed.. seems that on debian it's not so easy to install all so I just didn't do it hehe.